cumulusci/tasks/create_package_version.py

```diff
@@ -198,7 +198,7 @@ def _init_options(self, kwargs):
             or self.project_config.project__package__namespace,
             version_name=self.options.get("version_name") or "Release",
             version_base=self.options.get("version_base"),
-            version_type=self.options.get("version_type") or "minor",
+            version_type=self.options.get("version_type") or "patch",
```

Comment:
This should default to `build` to match the previous behavior. (When building a feature test package we want to keep the same major/minor/patch numbers, and only increment the build number by setting it to NEXT)

The release_2gp_beta flow should use this task with version_type set to `minor` though.

Reply:
done